recipes/gtest/all/conanfile.py

```diff
@@ -105,17 +119,26 @@ def generate(self):
         tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
         tc.variables["BUILD_GMOCK"] = bool(self.options.build_gmock)
         tc.variables["gtest_hide_internal_symbols"] = bool(self.options.hide_symbols)
+
+        if self.settings.build_type == "Debug" and Version(self.version) < "1.12.0":
+            tc.cache_variables["CUSTOM_DEBUG_POSTFIX"] = str(self.options.debug_postfix)
+
         if is_msvc(self) or self._is_clang_cl:
             tc.variables["gtest_force_shared_crt"] = not is_msvc_static_runtime(self)
         if self.settings.os == "Windows" and self.settings.compiler == "gcc":
             tc.variables["gtest_disable_pthreads"] = True
         tc.generate()
 
     def _patch_sources(self):
+        internal_utils = os.path.join(self.source_folder, "googletest",
+                                      "cmake", "internal_utils.cmake")
+        if Version(self.version) < "1.12.0":
+            apply_conandata_patches(self)
```

Comment:
```suggestion
        apply_conandata_patches(self)
        if Version(self.version) < "1.12.0":
```
apply_conandata_patches does not crash in case there is no patches to be applied.